HoloLensCommander/HoloLensCommander/project.json

```diff
@@ -2,6 +2,7 @@
   "dependencies": {
     "Microsoft.NETCore.UniversalWindowsPlatform": "5.1.0",
 	"WindowsDevicePortalWrapper": "0.9.4"
```

Comment:
This should be 0.9.5.1 to take advantage of updates and fixes (looks like _**I**_ may have missed that in a checkin last month-ish)

Reply:
@davidkline-ms Sounds good. Is there a cadence for updating dependency versions?